src/util/base58.rs

```diff
@@ -13,23 +13,22 @@ use crate::hashes::{sha256d, Hash};
 use crate::prelude::*;
 use crate::util::endian;
 
-/// An error that might occur during base58 decoding
+/// An error that might occur during base58 decoding.
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
 #[non_exhaustive]
 pub enum Error {
-    /// Invalid character encountered
+    /// Invalid character encountered.
     BadByte(u8),
-    /// Checksum was not correct (expected, actual)
+    /// Checksum was not correct (expected, actual).
     BadChecksum(u32, u32),
-    /// The length (in bytes) of the object was not correct
-    /// Note that if the length is excessively long the provided length may be
-    /// an estimate (and the checksum step may be skipped).
+    /// The length (in bytes) of the object was not correct, note that if the length is excessively
+    /// long the provided length may be an estimate (and the checksum step may be skipped).
```

Comment:
This is too long, a summary should be short and anything else it's own paragraph.

Reply:
Fixed as separate patch.